notifications_utils/jinja_templates/email_template.jinja2

```diff
@@ -28,39 +30,36 @@
 {% endif %}
 <span style="display: none;font-size: 1px;color: #fff; max-height: 0;">{{ preheader }}…</span>
 
+<div style="height:20px">&nbsp;</div>
```

Comment:
Some email clients don't support margins, this does the trick

Reply:
This was a trick from the 90s.. 😅